### PR TITLE
Clarify configuration via UI vs file

### PR DESCRIPTION
**Pull request by AndrewMorsillo**
Clarify that GUI configuration operations are not hidden away in a binary database.

Signed-off-by: Andrew Morsillo <imagio@protonmail.com> (github: AndrewMorsillo)

### CODE REVIEW

File: configuration/index.md [hunk 1]
```diff
@@ -133,5 +133,6 @@ Every interface has it's own strong points.
 With openHAB 1.x the smart home configuration was done via configuration files only.
 One of the most visible additions to openHAB 2 is the adminstrative web interface Paper UI.
 
-Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database.
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
+Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database. The database used by Paper UI is a human readable (and editable for advanced users) [JSON database]({{base}}/administration/jsondb.html) found in the `userdata` folder of your openHAB installation.
```
Comment (Confectrician) [inline configuration/index.md]: Please make a newline here.
We keep the docs to one sentence per line, to make maintaining them easier.

File: configuration/index.md [hunk 2]
```diff
@@ -133,5 +133,6 @@ Every interface has it's own strong points.
...
+
+Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file. Configuration done in files will be synced to the database but configuration done in the database will not be written back to files.
```
Comment (Confectrician) [inline configuration/index.md]: Please one sentence per line too here.

**Comment by ThomDietrich** [comment]
I have to be honest, I am a bit skeptical. This is the introductional article. The jsondb is not supposed to be edited manually. Yes it is human readable, that helps with backups and troubleshooting, but it is not a user-facing element I'd promote here. The end user ought to configure his system in config files or via Paper UI.

For troubleshooting or for users with the interest in technical details the link is a good idea! Hence I'd suggest to reduce your contribution to:

> Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
> Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
> Configuration done in files will be synced to the database but configuration done in the database will not be written to files.

File: configuration/index.md (hunk 2 above)
Comment (ThomDietrich) [reply configuration/index.md]: Please remove "back" in the second sentence.

**Comment by AndrewMorsillo** [comment]
@ThomDietrich I agree with the version you proposed. 

The reason I wanted to make this edit in the first place is that as a new OpenHAB 2 user I didn't know that the configuration data was not in a binary database. I wanted to set up alexa which requires tagging and that is not yet supported in paper UI. Once I learned that the DB was actually just JSON the easiest way to for me to add tags was to edit it directly. It is also nice to know that I can read the jsondb to help me understand what the UI is doing and why it is doing it. I agree that this is an introductory article and shouldn't promote advanced things like editing the DB. I will update the pull request to the version you proposed. Thanks for helping me out with my first openHAB contribution, hopefully there is a lot more I can contribute once I get time to really dig in and learn more about the internals!

**Comment by ThomDietrich** [comment]
@AndrewMorsillo interesting! It is sad that Paper UI is still not full-featured... that is btw one of many reasons why my setup is (still) completely config file based. I take great comfort in the knowledge, that my setup 100% depends on what I write in hard files I can backup, restore and mass-edit: https://github.com/ThomDietrich/openhab-config

**Comment by Confectrician** [comment]
@ThomDietrich can you give a short feedback here?
I think the initial commit was changed and it looks good from my side.

Wdyt?
I would say LGTM.

**Comment by ThomDietrich** [comment]
@Confectrician Absolutely.

@AndrewMorsillo just out of curiosity, did you find the time to look into the other PR?

**Pull request merged**